commons/commands.py

```diff
@@ -267,7 +275,12 @@
 CMD_SW_UP = "provisioner sw_upgrade --offline"
 CMD_SPACE_CHK = "df -h"
 CMD_FIND_FILE = "find /etc/cortx/ -name *.gz"
-SET_NAMESPACE = "kubectl config set-context --current --namespace={}"
+CMD_GET_ACCESS_KEY= " kubectl get pods | grep cortx-server- | cut -d ' ' -f1 | head -1 |" \
+                    " xargs -I{} kubectl exec -t {} -c cortx-rgw -- " \
+                    "cat /etc/cortx/cluster.conf | grep auth_admin | head -2 | " \
+                    "tail -1 | cut -d ':' -f2 | sed -e 's/^[[:space:]]*//'"
+CMD_GET_SECRET_KEY="cat /root/cortx-k8s/k8_cortx_cloud/solution.yaml |" \
```

Comment:
Similar function can be written to read solution.yaml. 
@pragamj please point out a function if already exists.